src/models/pdt.ts

```diff
@@ -82,7 +102,7 @@ export const genTestingRecord = (testingRecords: TestingRecord[], expiryDateTime
     const euTestRecord: EuTestRecord = {
       tg: targetDisease,
       tt: testingRecord.testTypeCode,
-      sc: testingRecord.collectionDateTime,
+      sc: dayjs.utc(testingRecord.collectionDateTime).format("YYYY-MM-DDTHH:mm:ss[Z]"),
```

Comment:
no need timezone converter for this UTC date. both input date and eu offline qr collectionDateTime date need to be UTC date. Shouldn't need to use dayjs timezone plugin to convert.
Note : EU required date format is like this
<img width="636" alt="Screenshot 2022-05-15 at 10 11 26 PM" src="https://user-images.githubusercontent.com/10098074/168477150-114306b7-e1b4-4982-b924-a68f661f5a84.png">
;